openreview/conference/builder.py

```diff
@@ -827,7 +835,7 @@ def create_blind_submissions(self, hide_fields=[]):
 
                 blind_note = self.client.post_note(blind_note)
 
-                if self.submission_stage.public:
+                if self.submission_stage.public and blind_content['_bibtex'] is None:
```

Comment:
I think what you need to check here is that `venue` doesn't exist. When `venue` exists, it means the post decision stage was ran and all the submissions were marked with a venue value and a bibtex entry including the rejected submissions. 

When the bibtex was set during the post decision stage we shouldn't override it if create_blind_submissions is called after that. Does it make sense?

Reply:
So what I did was always set the bibtex to the bibtex set in post_decision if it exists, therefore if there is already a bibtex in `blind_content` it will not be overwritten (only calculate bibtex if it is None), is this wrong?

Reply:
I do this earlier so that if nothing is being changed in the blind submission, then we won't even get this far because of this line:

`if not existing_blind_note or existing_blind_note.content != blind_content or existing_blind_note.readers != blind_readers:`

Reply:
If we ran create_blind_submissions twice and there was a change in the authors list then the bibtex is not being updated. I'm confused about this, let's have a call.